### ISSUE TITLE
Fix travis passing failed builds

### ISSUE THREAD

**Issue by pdehaan** [mozilla/fxa-content-server#1030]
**tl;dr:** Travis fails to create tunnel to sauce labs. Tests don't run. Travis still passes. Peter flips tables.
### Steps to reproduce:
1. Find passing travis builds that pass in < 6m
2. Wait, what???
### Actual results:

fxa-content-server Travis builds typically take 15-19 minutes for a full run.
Occasionally we see some Travis builds that pass in 5-6 minutes (LIES) that are actually failed builds:

> fxa-content-server@0.11.0 test-remote /home/travis/build/mozilla/fxa-content-server
> intern-runner config=tests/intern_sauce
> Defaulting to "runner" reporter
> Listening on 0.0.0.0:9090
> Missing Sauce Connect local proxy, downloading dependency
> This will only happen once.
> Downloading 3.02MB
> Unzipping sc-4.1-linux.tar.gz
> Setting execute permissions on /home/travis/build/mozilla/fxa-content-server/node_modules/intern-geezer/node_modules/sauce-connect-launcher/sc/sc-4.1-linux/bin/sc
> Removing sc-4.1-linux.tar.gz
> Sauce Connect installed correctly
> Opening local tunnel using Sauce Connect
> Starting sc with args: -u fxa-content -k ee5354a4-3d5e-47a0-84b0-0b7aaa12a720 -P 4445 --tunnel-identifier 1398786940593 --readyfile /tmp/sc-launcher-readyfile
> Creating tunnel with Sauce Labs
> **[Error: 29 Apr 15:57:41 - Error: failed to bring up tunnel VM.**
> 29 Apr 15:57:41 - Successfully freed.
> 29 Apr 15:57:41 - Cleaning up.]
> Removing /home/travis/build/mozilla/fxa-content-server/node_modules/intern-geezer/node_modules/sauce-connect-launcher/sc/sc-4.1-linux.tar.gz
> The command "npm run-script test-remote" exited with 0.
> **Done. Your build exited with 0.**
### Expected results:

Sauce tunnel fails, so does build.
This is risky risky since we may see a green Travis and merge a change, not seeing that Travis took a "close enough" approach to passing a build.

![](http://www.greenerpeople.com/forum/members/vbtroubleshooter-albums-threads-picture294-kermit.jpg)
